media/js/src/form-components/DefineRange.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
```

Comment:
Just a minor point, but because this file is using JSX syntax, make the filename `DefineRange.jsx`.